syncthing.if

```diff
@@ -0,0 +1,55 @@
+########################################
+##	<summary>
+##	Role access for Syncthing
+##	</summary>
+##	<param name="role">
+##		<summary>
+##			Role allowed access
+##		</summary>
+##	</param>
+##	<param name="domain">
+##		<summary>
+##			User domain for the role
+##		</summary>
+##	</param>
+#
+interface(`syncthing_role', `
+
+	gen_require(`
+		attribute_role syncthing_roles;
+		type syncthing_t, syncthing_exec_t, syncthing_config_home_t;
+	')
+
+	# Declarations
+	roleattribute $1 syncthing_roles;
+
+	# Policy
+	domtrans_pattern($2, syncthing_exec_t, syncthing_t)
+')
```

Comment:
@doverride so should I do this:

```
role $1 types syncthing_config_home_t;
userdom_user_home_dir_filetrans($2, syncthing_config_home_t, dir, "syncthing")
```

Since `syncthing_config_home_t` has the `user_home_content` attribute, it should be able to be r/w by the user.